### PR TITLE
Add historic changelog

### PR DESCRIPTION
**Pull request by t-b**
And get the documentation docker containers work again.

Required as first step for #346.

### CODE REVIEW

File: docu/sphinx/source/changelog.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+- EvaluateRTE: Avoid Igor crash due to wrong printf usage
+- Avoid passing CHECK_WAVE(..., NULL_WAVE) assertion for existing wave. This required to change the value of NULL_WAVE.
+- Documentation/Readme.md: minor style fixes
+
+1.07 (09/17/2018)
+-----------------
+
+- Convert documentation to sphinx and extend it a lot!
+- Allow selecting test cases and test suites using a regular expression as paramter to RunTest.
```
Comment (MichaelHuth) [inline docu/sphinx/source/changelog.rst:21]: typo: -> parameter

File: docu/sphinx/source/changelog.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  TestBegin, TestEnd, TestSuiteBegin, TestSuiteEnd, TestCaseBegin and TestCaseEnd were split into an internal function
+  that is always executed and a hookable function where a user can extend functionality. The User functions are called
+  directly after the internal functions.
+
+- More detailed error message on unexpected runtime errors
+- FIX: AbortFlag was not initialized on TEST_BEGIN
+- FIX: Check for Procedure File Names was Case-Sensitive
+- Docu: Use tabwidth of 4 for igor pro example code
+- Nicify examples
+- TEST_BEGIN/TEST_END: Turn off Igor Pro Debgger during test execution Turning off the debugger allows us to support
```
Comment (MichaelHuth) [inline docu/sphinx/source/changelog.rst:54]: typo: -> Debugger

**Review by MichaelHuth — CHANGES_REQUESTED** [review]
Just a few typos.

**Comment by t-b** [comment]
@MichaelHuth Thanks. Fixed.

**Review by MichaelHuth — APPROVED** [review]
Good

**Pull request merged**